src/app/core/services/split-expense.service.spec.ts

```
@@ -755,20 +739,33 @@ describe('SplitExpenseService', () => {
-          expect(splitExpenseService.postComment).toHaveBeenCalledWith(violationComment5);
-        });
+  it('postSplitExpenseComments(): should post split expense comments using expenseCommentService', (done) => {
+    const txnIds = ['txn1', 'txn2'];
+    const comments = {
+      0: 'First reason',
+      1: '',
+    };
+    const expectedPayload = [
+      {
+        id: 'txn1',
+        comment: splitExpenseService.prependPolicyViolationMessage + 'First reason',
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)